modules/cluster-logging-collector-log-forwarding-about.adoc

```diff
@@ -18,7 +18,7 @@ Forwarding cluster logs to external third-party systems requires a combination o
 
 * `kafka`. A Kafka broker. The `kafka` output can use a TCP or TLS connection.
 
-* `default`. The  internal {product-title} Elasticsearch instance using `default` in the pipeline. You do not need to configure the default output. If you do configure a `default` output, you receive an error message because the `default` output is reserved for the Cluster Logging Operator.
+* `default`. The internal {product-title} Elasticsearch instance. You do not need to configure the default output. If you do configure a `default` output, you receive an error message because the `default` output is reserved for the Cluster Logging Operator.
```

Comment:
```suggestion
* `default`. The internal {product-title} Elasticsearch instance. You must not configure the default output. If you do configure a `default` output, you receive an error message because the `default` output is reserved for the Cluster Logging Operator.
```

Reply:
I like the _You are not required to configure _ wording you suggested later